spec/index.html

```diff
@@ -495,6 +496,26 @@ <h2>RDF Documents and Syntaxes</h2>
     and still the subject of group discussion. An alternative to conformance
     levels, "profiles", may be adopted instead, abandoned, or described in 
     another specification.</p>
+
+  <section id="rdf-strings">
+    <h2>Strings in RDF</h2>
+
+    <p>RDF uses Unicode [[Unicode]] as the fundamental representation for string values.
+      Within this, and related specifications, the term <dfn id="dfn-rdf-string">string</dfn>
+      is used to describe an ordered sequence of zero or more <a data-lt="code point" class="lint-ignore">Unicode code points</a>
+      in any <a data-cite="i18n-glossary#dfn-character-encoding">character encoding</a>.
+      Note that most <a>concrete RDF syntaxes</a> require the use
+      of the UTF-8 character encoding [[!RFC3629]], 
+      which also prevents the use of certain non-character values.
+      are restricted to the UTF-8 character encoding [[!RFC3629]].
+      Strings are further restricted to those matching the
+      <a data-cite="XML11#charsets">Char</a> production [[XML11]].</p>
```

Comment:
See comment on https://github.com/w3c/rdf-concepts/issues/51#issuecomment-1707886347

Lexical forms in the abstract data model are not limited to displayable text. Just because thios WG doesn't see a usage for certain code points, it doesn't mean they should be excluded from the abstract data model. Discouragement or advice to avoid certain forms can be given as it is current - use normal form C.

```suggestion
</p>
```

Reply:
Turtle:
```
[39]  STRING_LITERAL_QUOTE  ::=  '"' ([^#x22#x5C#x0A#x0D] | ECHAR | UCHAR)* '"'
```

Or use  `\u` or `\U` escapes.

```
:subject :predicate "\u0000" 
```

Reply:
XML Char production does not include U+0000, the "noncharacters" FFFE, FFFF ("noncharacters").
It advises against certain other characters which we would effectively inherit - I think makes it not suitable for RDF Concepts.